packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -143,6 +144,7 @@ class RenderEditable extends RenderBox {
     Color backgroundCursorColor,
     ValueNotifier<bool> showCursor,
     bool hasFocus,
+    @required List<LayerLink> handleLayerLinks,
```

Comment:
Since we always require two: would it be more meaningful to just have to separate parameters with meaningful names instead of a list?

Reply:
(e.g. leading/trailingHandleLayerLink)?

Reply:
same elsewhere where the list is used.